workloads/kube-burner/run.sh

```diff
@@ -2,7 +2,7 @@
 
 . common.sh
 
-remove_update_taint
+#remove_update_taint
```

Comment:
Why did you remove this?, this issue is still present in latest builds

Reply:
took out during my own testing, adding back